docs/how-tos/howto-run-loadbot.md

```diff
@@ -0,0 +1,80 @@
+---
+id: howto-run-loadbot 
+title: How to run the loadbot for stress testing your network
+---
+
+## Prerequisites
+
+This how-to assumes you have followed
+the [guide on how to set up an IBFT cluster locally](/docs/how-tos/howto-setup-ibft/howto-set-ibft-locally)
+or [guide on how to set up an IBFT cluster on the cloud](/docs/how-tos/howto-setup-ibft/howto-set-ibft-on-the-cloud).
+The loadbot also works when using dev mode.
+
+A functioning node is required to stress test the network.
+
+When building a network using the Polygon SDK, you may want to test how many transactions your configuration can handle
+per second, or how your network will react when you submit valid and invalid transactions.
+
+The Polygon SDK binary includes a `loadbot` command, responsible for sending a specific number of transactions per
+second.
+
+This _how-to_ will explain how to run the loadbot on a single node configuration using 2 accounts prefilled with a
+balance of 1000 ETH each.
+
+:::note
+
+This is not the only parameter you can specify when running the loadbot.
+
+:::
+
+## Step 1: Create a genesis file with a premined account
+
+To generate a genesis file, run the following command:
+
+````bash
+go run main.go genesis --premine 0x1010101010101010101010101010101010101010 --premine 0x1010101010101010101010101010101010101020
+````
+
+The **premine** flag sets the address that should be included with a starting balance in the **genesis** file.<br />
+In this case, the addresses `0x1010101010101010101010101010101010101010` and
+`0x1010101010101010101010101010101010101020` will have a starting **default balance** of
+`0x3635C9ADC5DEA00000 wei`.
+
+## Step 2: Start the Polygon SDK in dev mode
```

Comment:
Does it need to start in dev mode?

Reply:
@PtitLuca I don't think that people who want to stress test the network need instructions on generating the genesis file and everything else.

Simply add as requirements to the top of the doc that you should have:
- A working Polygon-SDK network
- JSON-RPC reachable
- gRPC reachable
- Private key of the account with X tokens to use for testing
- anything else I may have missed